geoapps_utils/numerical.py

```diff
@@ -211,36 +296,39 @@ def find_curves(  # pylint: disable=too-many-locals
     if not hasattr(tri, "simplices"):
         return []
 
+    simplices: np.ndarray = getattr(tri, "simplices")
+
     edges = np.vstack(
         (
-            tri.simplices[:, :2],  # pylint: disable=no-member
-            tri.simplices[:, 1:],  # pylint: disable=no-member
-            tri.simplices[:, ::2],  # pylint: disable=no-member
+            simplices[:, :2],
+            simplices[:, 1:],
+            simplices[:, ::2],
```

Comment:
**suggestion:** `getattr` to circumvent pylint errors is not the right approach

Please see how Delaunay is declared in geoapps pylintrc:

```
# List of members which are set dynamically and missed by pylint inference
# system, and so shouldn't trigger E1101 when accessed. Python regular
# expressions are accepted.
generated-members=scipy.spatial._qhull.Delaunay.*
```